ADAL/src/ADAL_Internal.h

```diff
@@ -54,10 +54,12 @@
 typedef void(^ADAuthenticationCallback)(ADAuthenticationResult* result);
 
 
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IOS
 //iOS:
 #   include <UIKit/UIKit.h>
 typedef UIWebView WebViewType;
+#elif TARGET_OS_WATCH
+typedef NSString WebViewType; // Should never be called
```

Comment:
Should this be `id`? Giving it an actual type doesn't make much sense to me. Importantly, I think the accessor should be overridden to throw an exemption if called from on Apple Watch.